examples/static-site-gen/input/basics.md

```diff
@@ -0,0 +1,295 @@
+Getting the Gist of Markdown's Formatting Syntax
```

Comment:
Since we're putting this in the repo, can we link to its license somewhere? https://daringfireball.net/projects/markdown/license

(e.g. at the top of basics.md seems fine!)

Reply:
Oh, never mind - I see the license is included later on! 💯